### PR TITLE
improve label

### PR DESCRIPTION
**Pull request by tpiperatgod**
to fix issue #352 

Signed-off-by: laminar <tpiperatgod@gmail.com>

### CODE REVIEW

File: src/main/java/io/functionmesh/compute/util/CommonUtil.java [hunk 1]
```diff
@@ -78,7 +78,8 @@ public class CommonUtil {
     public static final String CLUSTER_LABEL_CLAIM = "pulsar-cluster";
     public static final String TENANT_LABEL_CLAIM = "pulsar-tenant";
     public static final String NAMESPACE_LABEL_CLAIM = "pulsar-namespace";
```
Comment (freeznet) [inline src/main/java/io/functionmesh/compute/util/CommonUtil.java:80]: do we need to update above labels as well?

**Review by freeznet — APPROVED** [review]

File: src/main/java/io/functionmesh/compute/util/CommonUtil.java (hunk 1 above)
Comment (tpiperatgod) [reply src/main/java/io/functionmesh/compute/util/CommonUtil.java:80]: according to the original need, only "pulsar components" were mentioned, but I think we can update these as well...

Comment (freeznet) [reply src/main/java/io/functionmesh/compute/util/CommonUtil.java:80]: yes, please update these labels to have the prefix as well.

**Review by jiangpengcheng — APPROVED** [review]

**Comment by tpiperatgod** [comment]
I think we can remove the deprecated labels after one or two releases.

**Review by freeznet — APPROVED** [review]

**Pull request merged**